homeassistant/components/geocaching/sensor.py

```diff
@@ -69,13 +69,14 @@ async def async_setup_entry(
     async_add_entities: AddConfigEntryEntitiesCallback,
 ) -> None:
     """Set up a Geocaching sensor entry."""
-    coordinator = hass.data[DOMAIN][entry.entry_id]
+    coordinator: GeocachingDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
```

Comment:
Hmm ideally, we migrate this integration to use the runtime_data in the config entry first, that way we can actually correct type it.